scripts/generate_errors.pl

```diff
@@ -60,11 +60,11 @@
 my $error_format = <FORMAT_FILE>;
 close(FORMAT_FILE);
 
-my @files = <$include_dir/*.h>;
+my @files = glob qq("$include_dir/*.h");
 my @necessary_include_files;
 my @matches;
 foreach my $file (@files) {
-    open(FILE, "$file");
+    open(FILE, '<:crlf', $file) or die("$0: $file: $!");
```

Comment:
There's a change here that's not documented: the addition of `<:crlf`. This has been around since Perl 5.8 which is ancient enough that it's not a problem to require it.

Reply:
Oh yes, nice spot - the `:crlf` was added in PR 4395 which is in `development` but not 2.28. The commit message says 

```
Accept Windows line endings in input files on any platform. This makes
the scripts work even when running a Unix perl with a source tree that
has Windows line endings, as happens for example on our Travis Windows
instances. This change is harmless in the common case where the input
has the platform's default line endings
```

and it seems reasonable (preferable) to do this.